src/main/java/uc/user/login/LoginInteractor.java

```diff
@@ -11,22 +11,22 @@
 public class LoginInteractor implements LoginInputBoundary {
     private StateTracker stateTracker;
     private LoginDsGateway dsGateway;
-    private LoginOutputBoundary presenter;
+    private LoginOutputBoundary outputBoundary;
```

Comment:
I think we should choose a convention here as some use cases are using outputBoundary and others are using presenter. Personally I would favour calling it presenter (obviously the type is still the output boundary)